install_config/topics/ocp_osp_enabling_octavia.adoc

```diff
@@ -111,3 +111,10 @@ backend is ML2/OVS. There is no need for modifications if the backend is
 ML2/OVN.
 ====
 
+[NOTE]
+====
+If Kuryr SDN is used, ML2/OVS is the preferred Neutron backend at the moment.
```

Comment:
We should remove `at the moment.` If the recommendation changes then the docs should be updated. So what about:

> ...ML2/OVS is the preferred Neutron backend.

Reply:
Agree